.github/workflows/build_docker_nightly.yml

```diff
@@ -1,6 +1,3 @@
-# NOTE: A custom version of this template workflow is used in neurobagel/query-tool.
```

Comment:
This comment is from the template version of this wf, synced from https://github.com/neurobagel/workflows/blob/main/template_workflows/build_docker_nightly.yml.

With this PR, this repo now also deviates from the synced template, meaning we probably should remove it from the sync.yml section for this file. I'd also suggest adding a comment about it in the template, similar to what we did for the query tool, so we don't forget.

Could you create an issue for this in https://github.com/neurobagel/workflows/issues?